src/ResourceManager/Network/ChangeLog.md

```diff
@@ -34,6 +34,8 @@
 * Add support for DDoS protection plan resource
 * Introduced multiple breaking changes
     - Please refer to the migration guide for more information
+* Added cmdlet to create protocol configuration
```

Comment:
@junyezhu please move the snippet you're adding to be under the `## Current Release` header

Reply:
@cormacpayne Done